.circleci/config.yml

```diff
@@ -202,7 +202,7 @@ aliases:
     api-cpu-threshold: 30
     api-memory-limit: 2Gi
     rpc-url: http://user:password@dogecoin-indexer-svc.unchained.svc.cluster.local:8332
-    api: false
+    api: true
```

Comment:
Refer to the other chains for full api configuration which are all the same at this time I am pretty sure